src/net/sourceforge/kolmafia/AdventureResult.java

```diff
@@ -1475,4 +1472,18 @@ public static WildcardResult getInstance(String text) {
       return null;
     }
   }
+
+  private int safeDowncastLongToInt(long value) {
+    int retValue;
+    try {
+      retValue = Math.toIntExact(value);
```

Comment:
Just return here

Reply:
Spaghetti code.  I have a distinct preference for code segments that have only one exit path.  Do I need to abandon that preference?